udata_hydra/db/resource_exception.py

```diff
@@ -0,0 +1,80 @@
+import json
+
+from asyncpg import Record
+
+from udata_hydra import config, context
+from udata_hydra.db.resource import Resource
+
+
+class ResourceException:
+    """Represents a row in the "resources_exceptions" DB table
+    Resources that are too large to be processed normally but that we want to have anyway"""
+
+    @classmethod
+    async def get_all(cls) -> list[Record]:
+        """
+        Get all resource_exceptions
+        """
+        pool = await context.pool()
+        async with pool.acquire() as connection:
+            q = "SELECT * FROM resources_exceptions;"
+            return await connection.fetch(q)
```

Comment:
Do we (want to) have a real ORM? Not sure how useful it would be, as it seems that at least some of the tables are generated dynamically...

Reply:
This is a good question... We considered it at some point, but we went that way as that part was already without ORM and that way we can make sure performances are good.
Also, I didn't want to mix several refactorings ate the same time.
But the question still stands!